codegen/src/test/java/org/web3j/codegen/ContractJsonParseTest.java

```diff
@@ -69,11 +69,11 @@ public void testParseConvertLib() throws Exception {
         Contract mc = parseContractJson(contractBaseDir, "MetaCoin", "ConvertLib");
 
         assertEquals("Unexpected contract name", "ConvertLib", mc.getContractName());
-        assertEquals("Unexpected number of functions", 1, mc.abi.size());
+        assertEquals(1, mc.abi.size());
         AbiDefinition abi = mc.abi.get(0);
         assertEquals("Unexpected function name", "convert", abi.getName());
-        assertTrue("Expected function to be 'constant'", abi.isConstant());
-        assertFalse("Expected function to not be 'payable'", abi.isPayable());
+        assertTrue(abi.isConstant());
+        assertFalse(abi.isPayable());
```

Comment:
Does junit 5 not support reasons?

Reply:
For whatever reason in JUnit 5 the second argument is the reason. Fixed in the latest commit I did.